### DISCUSSION TITLE
Is it possible to flash in Windows OS?

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/meta-tegra#1179] [Q&A] [answered]
Hi. I'm wondering if it's possible to do flash with host machine in Windows, or it can only be done in Linux?

**Comment by madisongh** [comment, marked as answer]
NVIDIA only supports flashing from Linux.  You can search around the developer forum for posts on this topic... it looks like some people have had some success with using a Linux VM under Windows, but even that's not really supported.
